### PR TITLE
init

### PR DESCRIPTION
**Pull request by yuichi10**
## Context
Add initial contents.

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+cover.html
+
+# plugins
+vendor/
+.vscode/
+
```
Comment (sugarnaoming) [inline .gitignore]: I want `.idea/` because i use goland.

Comment (yuichi10) [reply .gitignore]: I fixed

Comment (sugarnaoming) [reply .gitignore]: thx

File: screwdriver.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    environment:
+        GOPATH: /sd/workspace
+
+jobs:
+    main:
+        requires: [~pr, ~commit]
+        steps:
+            - install: go mod download
+            - vet: go vet ./...
+            - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
```
Comment (tk3fftk) [inline screwdriver.yaml]: how about more simplify like meta-cli?
https://github.com/screwdriver-cd/meta-cli/blob/9cfbdc987b7c55e6a03559f0d793d1f321124fa4/screwdriver.yaml#L15

File: screwdriver.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+shared:
+    image: golang
+    environment:
+        GOPATH: /sd/workspace
+
+jobs:
+    main:
```
Comment (tk3fftk) [inline screwdriver.yaml]: I think `main` may not be a good name.

File: screwdriver.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+        requires: [~pr, ~commit]
+        steps:
+            - install: go mod download
+            - vet: go vet ./...
+            - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - gover: go version
```
Comment (tk3fftk) [inline screwdriver.yaml]: I think showing version should be placed at first step or before using `go`.

File: screwdriver.yaml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+shared:
+    image: golang
```
Comment (tk3fftk) [inline screwdriver.yaml]: Do we need to specify docker tag?

File: screwdriver.yaml (hunk 2 above)
Comment (tk3fftk) [reply screwdriver.yaml]: And there is a difference between makefile's `gofmt` and screwdriver.yaml's `gofmt` command (`-s` option) .

Comment (yuichi10) [reply screwdriver.yaml]: I add s option to screwdriver.yaml.

File: screwdriver.yaml (hunk 3 above)
Comment (yuichi10) [reply screwdriver.yaml]: I fixed

File: screwdriver.yaml (hunk 4 above)
Comment (yuichi10) [reply screwdriver.yaml]: make sense. i fixed

File: screwdriver.yaml (hunk 5 above)
Comment (yuichi10) [reply screwdriver.yaml]: I add current latest version.

**Pull request merged**